packages/project/test/fixtures/component.a/src/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "_version": "1.1.0",
+    "sap.app": {
+        "_version": "1.1.0",
+        "id": "id1",
+        "type": "application",
```

Comment:
Should this be a `component`?

I understand that it's an `UI5/cli` implementation for components, but I was wondering how this would affect the manifest.json?

If my assumption is correct, please adjust also the values for the other test fixtures

Reply:
Thank you for the explanation @codeworrior !
I assumed something like that, but wanted to be sure